### DISCUSSION TITLE
ICC support?!?! or GPT-4 hallucination?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1138] [Q&A] [answered]
GPT-4 just told me "You can adapt this code to handle other color spaces or perform more advanced color management operations using the Python colour-science library, which provides extensive support for ICC profile aware color conversions and color management workflows."

Ummmm... have I overlooked the ICC-aware functions somehow?

- J

**Comment by thomasmansencal** [comment, marked as answer]
Ah brilliant :)

No, we unfortunately do not have any ICC related code!
